core/src/OpenMP/Kokkos_OpenMP_UniqueToken.hpp

```diff
@@ -22,26 +22,31 @@
 namespace Kokkos::Experimental {
 template <>
 class UniqueToken<OpenMP, UniqueTokenScope::Instance> {
+ public:
+  using execution_space = OpenMP;
+  using size_type       = int;
+
  private:
   using buffer_type = Kokkos::View<uint32_t*, Kokkos::HostSpace>;
-  int m_count;
+  execution_space const& m_exec;
```

Comment:
Why a const reference rather than a copy?  This implicitly deletes the copy and move assignment operators and makes it harder to reason about that type.

Reply:
I hesitated between copy and reference. I used reference because it seemed safer. This ways I know that the execution space was initialized correctly.